src/model-source/model-source.ts

```diff
@@ -58,4 +59,15 @@ export abstract class ModelSource implements IActionHandler {
     }
 
     abstract handle(action: Action): ICommand | Action | void;
+
+    /**
+     * Commit changes from the internal SModel back to the currentModel.
```

Comment:
Please add something like: "This method is meant to be called only by `CommitModelCommand` and other commands that need to feed the current internal model back to the model source."